modules/libcom/src/cvtFast/cvtFast.h

```diff
@@ -27,9 +30,7 @@
 extern "C" {
 #endif
 
-/*
- * All functions return the number of characters in the output
- */
+/** \brief All functions return the number of characters in the output */
```

Comment:
This `\brief` will only apply the comment to the first function below it. If you're not going to add a description of each one with a `\returns` line I suggest moving the comment into the file description at the top, preferably as a separate paragraph.